### ISSUE TITLE
Build test fails on Arch Linux

### ISSUE THREAD

**Issue by heartburn22** [dlenski/python-vipaccess#70]
I'm getting a failed test when trying to build on Arch Linux from the AUR. It was building fine about six weeks ago. 

======================================================================
FAIL: test_utils.test_check_TOTP_token_models:4
'SYDC', 'period', 'counter', False
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/mark/.cache/yay/python-vipaccess/src/python-vipaccess-0.14.2/tests/test_utils.py", line 131, in provision_valid_token
    assert check_token(test_otp_token, test_token_secret)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError

>>> assert check_token(test_otp_token, test_token_secret)

**Comment by dlenski** [comment]
> I'm getting a failed test when trying to build on Arch Linux from the AUR. It was building fine about six weeks ago.

_Consistently and repeatedly so_? Or only as a one-time flaky failure?

I already had to add 3-second delays in between the repeated invocations of this test in https://github.com/dlenski/python-vipaccess/commit/cc4366f7bce41d5ebce64ae8d86cc71e5eda5703, because the remote/web service appears not to like it if you call it too frequently.

**Comment by heartburn22** [comment]
Hi! Thanks for the reply.
I tried it several times last night, and I tried it again just now. I got the same result each time.

I did not notice a delay. In fact, I see this summary after the build fails:

----------------------------------------------------------------------
Ran 11 tests in 2.346s

FAILED (failures=2)
==> ERROR: A failure occurred in check().
    Aborting...
 -> error making: python-vipaccess-exit status 4
 -> Failed to install the following packages. Manual intervention is required:
python-vipaccess - exit status 4

**Comment by heartburn22** [comment]
Hi!
On a whim, I tried the install again today and it worked! It did not re-download the package, but the test passed this time. I'm not sure what changed.

**Comment by dlenski** [comment]
> I tried it several times last night, and I tried it again just now. I got the same result each time.

@heartburn22, are you certain that it was the _exact same_ test case failing each time? 

```
FAIL: test_utils.test_check_TOTP_token_models:4
'SYDC', 'period', 'counter', False
```

**Comment by heartburn22** [comment]
Yes. that is the version I was having trouble with, and it finally worked tonight. 
I'm sorry, I can't say for sure if it was failing on the exact same test each time.
